lib/commands.coffee

```diff
@@ -6,6 +6,7 @@ class Commands
     @_subscriptions.add atom.commands.add 'atom-workspace',
       'linter:next-error': @nextError.bind(@)
       'linter:toggle': @toggleLinter.bind(@)
+      'linter:set-bubble-transparent': @setBubleTransparent.bind(@)
```

Comment:
Stand alone `@` are discouraged in CoffeeScript. Instead, you can add a fat arrow to `setBubleTransparent` below and you don't have to bind it here.